### PR TITLE
Vet Clinic: Create Animals Table

### PR DESCRIPTION
**Pull request by raminka13**
## Animals Table
- Used [this template](https://github.com/microverseinc/curriculum-template-databases) to generate a new repo.
- Created a database named `vet_clinic`.
- Create a table named `animals` and copied the SQL code into the `schema.sql` file.
- Inserted 4 animals data into the `animals` table and copied the SQL code into the `data.sql` file. 
- Created several queries for the `animals` table and copied the SQL code into the `queries.sql` file.
- Took screenshots of the queries results.

Screen Shot 2022-09-12 at 14 46 11

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id,name, date_of_birth , neutered, escape_attempts, weight_kg) 
+VALUES (2,'Gabumon', '2018-11-15', 2, TRUE, 8);
+
+INSERT INTO animals (id,name, date_of_birth , neutered, escape_attempts, weight_kg) 
+VALUES (3,'Pikachu', '2021-01-07', 1, FALSE, 15.04);
+
+INSERT INTO animals (id,name, date_of_birth , neutered, escape_attempts, weight_kg) 
+VALUES (4,'Devimon', '2017-05-12', 5, TRUE, 11);
```
Comment (sja-thedude) [inline data.sql]: - [x] Kindly make sure that all the values are matched with the data. As you can see for escape_attempts you have displayed the value True which is not right. 
You have shuffled the values of neutered and escape_attempts. i.e neutered should be a boolean value and escape_attempts a numeric value.
```sql
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Agumon', '2020/2/3', '0', 'true', '10.23');
```

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND  '2019-01-01';
```
Comment (sja-thedude) [inline queries.sql]: - [ ] Kindly make sure that the end date should be 2019-31-12 as according to requirements it should display list of the names of all animals born between 2016 and 2019.
`SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';`

- [ ] Kindly make sure to add a screenshot for this query as in the pull request the query for this is not right.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @raminka13, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Correct gitflow
✔️ Professional Readme

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (raminka13) [reply data.sql]: Great suggestion, thanks. DONE

File: queries.sql (hunk 2 above)
Comment (raminka13) [reply queries.sql]: Makes a lot of sense, thanks. DONE

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Raminka, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
- professional looking readme file

## Required Changes ♻️
- [ ] it looks as if you have forgotten to push the changes you made to your GitHub repo.
if anything is not clear please don't hesitate to put a comment in the PR.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Raminka, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job following the previous instructions.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**